### ISSUE TITLE
SweepFormula: Add shortcuts for referencing special sweep subsets

### ISSUE THREAD

**Issue by t-b** [AllenInstitute/MIES#288]
The user wants to have shortcuts for referring to all available sweeps and all displayed sweeps. This is required for aumatically taking into account newly acquired sweeps.

The original proposal was

```
Allow for [] to mean all displayed sweeps, one can also use
[0...inf] to mean all available sweeps. (separate issue)
```

but on closer examination this looks really hacky.

How about having something like

```
data([100, 200],
 channels(AD),
 displayed())

data([100, 200],
 channels(AD),
 all())
```

which refers to all displayed sweeps (`displayed`) and all sweeps (`all`).

**Comment by ukos-git** [comment]
I did not see the new issue. I responded to this in https://github.com/AllenInstitute/MIES/issues/271#issuecomment-534161164

Lets keep the discussion for non-magic here.

**Comment by ukos-git** [comment]
ok. I now get a sweeps() list with all sweeps by calling `sweeps(all)` but as `PA_GetTraceInfos(graph, channelType = channelType)` only gets the currently plotted traces, there is no difference between the two calls to `data()`.

**Comment by t-b** [comment]
Note to self: use https://github.com/AllenInstitute/MIES/issues/271#issuecomment-534161164 but "allsweeps()" -> "all()" and "sweeps()" - > "displayed"

**Comment by ukos-git** [comment]
https://en.wikipedia.org/wiki/Syntactic_sugar

**Comment by t-b** [comment]
Note to self: Check that `all`/`displayed` really work and remove notice from help.

**Comment by timjarsky** [comment]
@t-b with the implementation of the `all` optional parameter for the `select` formula, can we now close this issue?

**Comment by t-b** [comment]
Yes indeed this was fixed in #1372.
